pkg/webhook/remotesecret_mutator.go

```diff
@@ -92,6 +97,7 @@ func (m *RemoteSecretMutator) CopyDataFrom(ctx context.Context, user authv1.User
 	}
 
 	if err := m.checkHasPermissions(ctx, user, sourceName, sourceNamespace); err != nil {
+		metrics.UploadRejectionsCounter.WithLabelValues(metricCopyDataDataOperationLabel, "insufficient_permissions").Inc()
```

Comment:
We possibly could distinguish between insufficient permissions and failure to check the permissions...

Reply:
ok